packages/app/src/app/overmind/namespaces/live/liveMessageOperators.ts

```diff
@@ -398,11 +401,10 @@ export const onConnectionLoss: Operator<LiveMessage> = mutate(
 
 export const onDisconnect: Operator<
   LiveMessage<{
-    reason: string;
+    reason: 'close' | 'inactivity';
```

Comment:
Would add a type here as well, like the rest of them :)

A bit confused that you can give it a dictionary type here? I believe this is the "event.type"?